nginx/etc/confd/templates/nginx/nginx_ip_whitelist_realip.conf.tmpl

```diff
@@ -1,7 +1,7 @@
-{{ if getenv "EXTERNAL_LOAD_BALANCER_IP" }}
+{{ if getenv "EXTERNAL_LOAD_BALANCER_HOST_IPS" }}
```

Comment:
TRUSTED_REVERSE_PROXIES should be allowed irrespective of whether there's an endpoint